### DISCUSSION TITLE
need help configure x-road node

### DISCUSSION THREAD

**Discussion by MarcosxDeveloper** [nordic-institute/X-Road#2436] [General]
hello!

i need implement x-road in my government organization using the next document: https://github.com/ogticrd/xroad-members/blob/master/docker-compose.yml

for security vulnerabilities policy, i want to know if have any problem from use the last version 7.51 for connect to another nodes on 7.2.2.

greetins

**Comment by petkivim** [comment]
Hi @MarcosxDeveloper! Please note that the Docker Compose setup is for test and development purposes only. It's a great way to get started with X-Road, but it cannot be used for a production level deployment. Also, the [Central Server](https://hub.docker.com/r/niis/xroad-central-server/) Docker image is not production ready and therefore, you should use the Central Server Debian packages for Ubuntu. Instead, the [Security Server Sidecar](https://hub.docker.com/r/niis/xroad-security-server-sidecar/) Docker image is designed for production use. 

Currently, the officially supported X-Road versions are 7.5.1, 7.4.2 and 7.3.2. If you use older versions, it's highly recommended to upgrade to one of the supported versions. Newer versions are backwards compatible with version 7.2.2 so you shouldn't have any issues with that. Also, please note that upgrades are supported from two previous versions which means that you cannot upgrade from 7.2.2 to 7.5.1 directly. Instead, you must upgrade from 7.2.2 to 7.3.2 or 7.4.2 first, and then to 7.5.1.
